test/runtests.jl

```diff
@@ -55,9 +55,9 @@ Mocking.activate()
         apply(get_patch) do
             try
                 AdventOfCode._setup_data_file(2019, 1)
-                @test isdir("data/2019")
-                @test isfile("data/2019/day_1.txt")
-                @test readlines("data/2019/day_1.txt") == ["TESTDATA"]
+                @test isdir("2019/data/")
+                @test isfile("2019/data/day_1.txt")
+                @test readlines("2019/data/day_1.txt") == ["TESTDATA"]
                 @test_logs (:warn, r"will not redownload") AdventOfCode._setup_data_file(2019, 1)
             finally
                 rm("data", recursive = true, force = true)
```

Comment:
We need to change this line too now to ensure that the test-folders are cleaned up
```suggestion
                rm("2019", recursive = true, force = true)
```